frontend/heart-code/src/components/profile-page/Profile.tsx

```diff
@@ -0,0 +1,35 @@
+import { User } from "../../models/User";
+import { Card, Button } from "react-bootstrap";
+import "./Profile.css";
+
+interface Props {
+  user: User;
+  isPersonalPage: boolean;
+}
+
+export default function Profile({ user, isPersonalPage }: Props) {
+  return (
+    <div className="profile-page">
+      <Card className="profile-card">
+        <Card.Img
+          variant="top"
+          src="https://optimaldataintelligence.com/wp-content/themes/optimaldataintelligence/images/image-not-found.png"
+        />
+        <Card.Body className="card-content">
+          <Card.Title>{user.alias}</Card.Title>
+          <Card.Subtitle className="card-content">Gender, age</Card.Subtitle>
+          <Card.Subtitle>Relationship type</Card.Subtitle>
+          <Card className="bio">
+            <Card.Text>Bio text goes here</Card.Text>
+          </Card>
+        </Card.Body>
+        {/* Needs to navigate to edit profile page later*/}
+        {isPersonalPage ? (
+          <Button className="edit-button">Edit</Button>
+        ) : (
+          <>{/*needs to return like and dislike buttons*/}</>
```

Comment:
ik denk dat het logischer is om dat in de page components te doen (de edit button in PersonalPage en de like/dislike buttons in de browsing page wanneer die gemaakt is), het profile component hoeft het dan alleen informatie te tonen

Reply:
zal ik dit dan nog even weg halen?

Reply:
de edit button zou ik in PersonalPage.tsx zetten, de like/dislike buttons weglaten

Reply:
ja precies, zal ik nog even doen

Reply:
dank je wel!